test/assert.js

```diff
@@ -792,13 +792,44 @@ describe('assert', function () {
 
     err(function() {
       assert.isAbove(null, 1);
-    }, 'expected null to be a number');
+    }, 'expected null to be a number or a date');
 
     err(function() {
       assert.isAbove(1, null);
     }, 'the argument to above must be a number');
   });
 
+  it('above (dates)', function() {
+    var now = new Date();
+    var oneSecondAgo = new Date(now.getTime() - 1);
+    assert.isAbove(now, oneSecondAgo, 'Now should be above 1 second ago');
+
+    err(function() {
+      assert.isAbove(oneSecondAgo, now);
+    }, 'expected ' + now.toUTCString() + ' to be above ' + oneSecondAgo.toString()); // TODO: Fix different date formatting
```

Comment:
One thing I noticed is that date formatting is not uniformal. While the first argument gets printed as a UTC string, second one is not. Probably should not be this way.